vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java

```diff
@@ -825,6 +830,31 @@ private void createGridWithExactItemCount() {
         addCard("Lazy Loading", "Exact item count", grid);
     }
 
+    private void createGridWithPagedRepository() {
+        //@formatter:off
+        // begin-source-example
+        // source-example-heading: Loading from paged repository
+
+        Grid<Person> grid = new Grid<>();
+        PersonService service = new PersonService();
+        /*
+         * For those backend repositories which use paged data fetching, it
+         * is possible to get the page number and page size from Query API.
```

Comment:
Instead of "backend repositories" it should just be "backends" IMO.

Also you could add a note about Spring:
"For backends which provide data in pages, like Spring Data repositories, it is possible to get the page number and size from the Query API."

Reply:
Maybe also add. "... For more instructions on how to work with Spring Data, see the documentation in vaadin.com/docs."

Reply:
Done. Also added a link to the tutorial. It includes an examples how to work with Spring backends